### DISCUSSION TITLE
Plerr and Pline in the REPCA1 Model

### DISCUSSION THREAD

**Discussion by sghqsads** [CURENT/andes#511] [Q&A]
Dear Andes developers,

I'm seeking insights into the REPCA1 model, particularly regarding the function and significance of setting Plerr (Plerr = Plant_pref - Pline), Plant_pref and Pline. **Why do we use ‘Plant_pref - Pline’ in this model instead of using Plant_pref alone?**  **For instance, in PVD1 model, there is no Pline.** What exactly does Plerr represent in this context and why do we need to use Pline? Also, is it feasible to set Pline to zero?

**Furthermore, if a bus is connected to multiple renewable generators, how should Plant_pref and Pline be set for each renewable generator?** Should the Plant_pref for each renewable generator be set as the sum of Plant_pref for all renewable generators?

Thank you!

**Comment by cuihantao** [comment]
Hello,

Some renewable plants connect to the grid by one single line. These plants accept signals from balancing authorities regarding their power output. That's why they control Plant_ref to Pline.

These REEC_A and REGC_A controlled by REPC_A are supposed to aggregated models that represent a large number of generators. 

> if a bus is connected to multiple renewable generators, how should Plant_pref and Pline be set for each renewable generator

I guess one single generator won't worry too much about the line power. But if you want to represent a group of individual generators, you can build a control loop that monitors the PCC output power and control it by an algorithm that adjusts the individual setpoints.

You can prototype this by modifying REPC_A to work with two REEC_A.
